packages/main/src/Link.ts

```diff
@@ -248,6 +290,10 @@ class Link extends UI5Element implements ITabbable {
 			&& this._isCrossOrigin();
 
 		this._rel = needsNoReferrer ? "noreferrer noopener" : undefined;
+		this.hasEndIcon = !!this.endIcon;
```

Comment:
Isn't this `this.hasEndIcon = !!this.endIcon;` always true, because slots are empty arrays by default?